### PR TITLE
libpam: Update to 1.3.0

### PR DESCRIPTION
**Pull request by neheb**
Removed upstreamed patches.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @nmav 
Compile tested: ipq806x

### CODE REVIEW

File: libs/libpam/patches/0007-Replace-strndupa-with-strndup.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+diff --git a/modules/pam_exec/pam_exec.c b/modules/pam_exec/pam_exec.c
+index 0ab6548..a003124 100644
+--- a/modules/pam_exec/pam_exec.c
++++ b/modules/pam_exec/pam_exec.c
+@@ -178,11 +178,11 @@ call_exec (const char *pam_type, pam_handle_t *pamh,
+ 		}
+ 
+ 	      pam_set_item (pamh, PAM_AUTHTOK, resp);
+-	      authtok = strndupa (resp, PAM_MAX_RESP_SIZE);
```
Comment (nmav) [inline libs/libpam/patches/0007-Replace-strndupa-with-strndup.patch]: Shouldn't in that case free the output?

Comment (nmav) [reply libs/libpam/patches/0007-Replace-strndupa-with-strndup.patch]: It may be easier to create a fixed buffer of pam_max_resp_size and copy there.

Comment (hnyman) [reply libs/libpam/patches/0007-Replace-strndupa-with-strndup.patch]: @neheb @nmav 
Any update on this PR?

Comment (neheb) [reply libs/libpam/patches/0007-Replace-strndupa-with-strndup.patch]: Not sure how to resolve this. My C knowledge is limited. A fix should probably be upstreamed first.

**Comment by champtar** [comment]
@neheb can you rebase on master to test CircleCI

**Comment by neheb** [comment]
The branch already is rebased on master

**Pull request merged**